src/components/shared/layout/Header.tsx

```diff
@@ -58,39 +61,19 @@ const Header = ({
           </li>
         )}
         {menu.map(({ label, url }) => (
-          <li
+          <NavItem
             key={url}
-            className={`fr-nav__item ${
-              currentMenu === url ? 'fr-nav__item-Active' : ''
-            }`}
-          >
-            <Link href={url} prefetch={false}>
-              <a
-                className="fr-nav__link"
-                aria-current={currentMenu === url ? 'page' : undefined}
-              >
-                {label}
-              </a>
-            </Link>
-          </li>
+            title={label}
+            current={currentMenu === url}
+            asLink={<a href={url}></a>}
```

Comment:
Should the `asLink`props value not be something like `<Link href  ...` ?

Reply:
it should be :/ but it's not working because Link does not allow an onClick property

Reply:
I don't see why we need an onClick ?
Also, With these links, we bypass the router and loose the construction of the unique layout (each click on menu will rebuild all the page content, so we loose the interrest of the SPA) and we can not use the Context update off the page (the content load in context after build of the layout will be lost at each rebuild).

Reply:
to close the menu when changing menu in mobile view